iceoryx_dust/include/iceoryx_dust/relocatable_pointer/relocatable_ptr.hpp

```diff
@@ -100,10 +104,14 @@ class relocatable_ptr
 
     /// @brief Convert to the corresponding raw pointer.
     /// @return corresponding raw pointer
+    // NOLINTJUSTIFICATION implicit conversion to raw pointer is intentional in design of relocatable structures
+    // NOLINTNEXTLINE(hicpp-explicit-conversions)
```

Comment:
Same as, above. Ok in the context of `dust`. Otherwise I would prefer explicit conversion.

Reply:
Same answer.